pkg/cstor/api_service.go

```diff
@@ -55,12 +55,12 @@ func (p *Plugin) httpRestCall(url, reqtype string, data []byte) ([]byte, error)
 
 	respdata, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
-		return nil, errors.Errorf("Unable to read response from maya-apiserver : %s", err.Error())
+		return nil, errors.Errorf("Unable to read response from maya-apiserver, err=%s data=%s", err.Error(), string(respdata))
```

Comment:
'm not sure whether respdata can be used in all cases when ioutil.ReadAll returns error

Reply:
can you verify this one from docs or code?

Reply:
Yes, It can be empty or partially filled in case of error, so it won't be useful in all the cases.

Reply:
my qn is: does it crash if we try use respdata when it is nil?

Reply:
ok. No, it won't crash.